content/support-matrix/delfin/dell-emc/vmax/_index.md

```diff
@@ -0,0 +1,363 @@
+      <tr>
+         <td rowspan=18>Device</td>
+         <td>Name</td>
+         <td>Device Name</td>
+      </tr>
+      <tr>
+         <td>Vendor</td>
+         <td>Vendor</td>
+      </tr>
+      <tr>
+         <td>Description</td>
+         <td>Device Description</td>
+      </tr>
+      <tr>
+         <td>Model</td>
+         <td>Device Model</td>
+      </tr>
+      <tr>
+         <td>Firmware version</td>
+         <td>Firmware version</td>
+      </tr>
+      <tr>
+         <td>Status</td>
+         <td>It shows availability of device ex: offline, normal etc.</td>
+      </tr>
+      <tr>
+         <td>Serial Number</td>
+         <td>It is VMAX array id</td>
+      </tr>
+      <tr>
+         <td>Location</td>
+         <td>Device Location, is name-value pair having component type and component name</td>
+      </tr>
+      <tr>
+         <td>Total Capacity</td>
+         <td>It is measured in terms of Tera byte</td>
+      </tr>
+      <tr>
+         <td>Used Capacity</td>
+         <td>It is measured in terms of Tera byte</td>
+      </tr>
+      <tr>
+         <td>Free Capacity</td>
+         <td>It is measured in terms of Tera byte</td>
+      </tr>
+      <tr>
+         <td>Raw Capacity</td>
+         <td>It is measured in terms of Giga byte</td>
+      </tr>
+      <tr>
+         <td>Subscribed Capacity</td>
+         <td>Refers to capacity allocated to device/host</td>
```

Comment:
Not allocated.. it is the total amount of capacity subscribed

Reply:
Done